app/frontend/src/components/__mocks__/mapbox-gl.js

```diff
@@ -0,0 +1,5 @@
+/// TODO: This mock may not be necessary if redux is removed.
+window.URL.createObjectURL = jest.fn();
+
+const mapboxgl = require("mapbox-gl");
```

Comment:
Is this because if you do `import`, it won't use the mock above?

Reply:
Yep